### PR TITLE
Added hasEffects() function

### PR DESCRIPTION
**Pull request by zKoz210**
## Introduction

I know you could do this in a plugin, but this function would be very handy for developers.

### API changes

`Living::hasEffects()` - Returns true if the entity has at least one effect

## Tests
Everything works, there is nothing to test

### CODE REVIEW

**Comment by dktapps** [comment]
> Everything works, there is nothing to test

Please don't take this attitude... if you want to contribute code, then test it first, always.

File: src/pocketmine/entity/Living.php [hunk 1]
```diff
@@ -232,6 +232,14 @@ public function getEffect(int $effectId){
 	public function hasEffect(int $effectId) : bool{
 		return isset($this->effects[$effectId]);
 	}
+	
+	/**
+	 * Returns whether the any effect is active on the mob.
+	 * @return bool
+	 */
+	public function hasEffects() : bool{
+		return count($this->effects) > 0;
```
Comment (dktapps) [inline src/pocketmine/entity/Living.php]: This could also be `!empty($this->effects)`

Comment (zKoz210) [reply src/pocketmine/entity/Living.php]: Yeah, I know, but what's better? To do as you have?

File: src/pocketmine/entity/Living.php [hunk 2]
```diff
@@ -232,6 +232,14 @@ public function getEffect(int $effectId){
 	public function hasEffect(int $effectId) : bool{
 		return isset($this->effects[$effectId]);
 	}
+	
+	/**
+	 * Returns whether the any effect is active on the mob.
```
Comment (Muqsit) [inline src/pocketmine/entity/Living.php]: Grammar?

Comment (zKoz210) [reply src/pocketmine/entity/Living.php]: @Muqsit I'd appreciate it if you fixed my mistake. I don't know english well.

Comment (Muqsit) [reply src/pocketmine/entity/Living.php]: Returns whether the mob has any active effects.

File: src/pocketmine/entity/Living.php [hunk 3]
```diff
@@ -238,7 +238,7 @@ public function hasEffect(int $effectId) : bool{
 	 * @return bool
 	 */
 	public function hasEffects() : bool{
-		return count($this->effects) > 0;
+		return !empty($this->effects);
```
Comment (KennFatt) [inline src/pocketmine/entity/Living.php]: checking an array with empty() is much way better than count it. In this case, we only check if empty or filled. 👍

**Review by KennFatt — APPROVED** [review]

**Review by dktapps — APPROVED** [review]

**Pull request merged**